src/main/java/onehundredandfour/Player.java

```diff
@@ -4,15 +4,15 @@
 import java.util.List;
 
 public class Player {
-	private final List<Card> hand;
+	private final List<?> hand;
 	private final List<Card> score = new ArrayList<>();
 	private Card cardOnTable;
 
-	public Player(final List<Card> hand) {
-		this.hand = hand;
+	public Player(final ArrayList<?> arrayList) {
```

Comment:
I think List is to be preferred here to maximize the choice of the implementation